recipe/meta.yaml

```
@@ -16,36 +18,28 @@ build:
     - pdfstream = pdfstream.main:main
     - run_server = pdfstream.main:run_server
     - print_server_config_dir = pdfstream.main:print_server_config_dir
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8, <3.10
     - pip
   run:
-    - python >=3.6
+    - python >=3.8, <3.10
     - bluesky
-    - databroker
+    - databroker =1.2
```

I think pining databroker like this should work.
```suggestion
    - databroker <2.0.0a
```